README.md

```diff
@@ -9,76 +9,247 @@ Low-level R library bindings
+- **MacOS/Linux**
+    ```bash
+    cargo build
+    ```
+- **Windows**
+    ```Shell
+    cargo build --target x86_64-pc-windows-gnu # 64-bit
+    cargo build --target i686-pc-windows-gnu   # 32-bit
+    ```
+
+
+To test the build, run `cargo test`. Note the `--test-threads=1`, without this flag `R` integration tests will fail:
+
+- **MacOs/Linux**
+    ```bash
+    cargo test -- --nocapture --test-threads=1
+    ```
+- **Windows**
+    **NOTE:** currently `Windows` tests are unstable, but they still can be executed with extra efforts:
+    <details>
+    <summary>Running tests on Windows</summary>
+
+    First, locate the installation of `R` and ensure that environment variable `R_HOME` points to it.
+    The standard value for the latest `R` version is `C:\Program Files\R\R-4.0.3`.
+
+    In order to run tests, `PATH` variable should be temporarily prepended with the path to correct `R.dll`.
+
+    - **64-bit**
+      - **CMD**
+        ```Shell
+        set OLD_PATH=%PATH%                        # Captures current PATH
+        set PATH=%R_HOME%\bin\x64;%PATH%           # Prepends with correct R directory
+        cargo test --target x86_64-pc-windows-gnu -- --nocapture --test-threads=1
+        set PATH=%OLD_PATH%                        # Resets PATH to the original value
+        ```
+      - **PowerShell**
+        ```PowerShell
+        $OLD_PATH=$env:PATH                        # Captures current PATH
+        $env:PATH="$env:R_HOME\bin\x64;$env:PATH"  # Prepends with correct R directory
+        cargo test --target x86_64-pc-windows-gnu -- --nocapture --test-threads=1
+        $env:PATH=$OLD_PATH                        # Resets PATH to the original value
+        ```
+    - **32-bit**
+      - **CMD**
+        ```Shell
+        set OLD_PATH=%PATH%                        # Captures current PATH
+        set PATH=%R_HOME%\bin\i386;%PATH%          # Prepends with correct R directory
+        cargo test --target i686-pc-windows-gnu -- --nocapture --test-threads=1
+        set PATH=%OLD_PATH%                        # Resets PATH to the original value
+        ```
+      - **PowerShell**
+        ```PowerShell
+        $OLD_PATH=$env:PATH                        # Captures current PATH
+        $env:PATH="$env:R_HOME\bin\i386;$env:PATH" # Prepends with correct R directory
+        cargo test --target i686-pc-windows-gnu -- --nocapture --test-threads=1
+        $env:PATH=$OLD_PATH                        # Resets PATH to the original value
+        ```
+
+
+    </details>
+
+
+## Building bindings from source (advanced)
+
+The bindings can be generated using [bindgen](https://github.com/rust-lang/rust-bindgen), special `rust` crate. 
+`bindgen` usage is enabled via `use-bindgen` feature flag.
+
+`bindgen` requires [libclang](https://clang.llvm.org/docs/Tooling.html), which should be installed first. 
+This library relies on `LIBCALNG_PATH` environment variable to determine path to the appropriate version of `libclang`.
```

Comment:
Please note the typo here: `LIBCLANG_PATH`

Reply:
Fixed in the followup commits